about.css

```diff
@@ -0,0 +1,527 @@
+  }
+
+  .header-menu-general {
+    display: flex;
+  }
+
+  .header-menu {
+    display: flex;
+    width: 100%;
+    justify-content: right;
+    background-color: var(--primary_4);
+    padding-right: 20px;
+    padding-bottom: 20px;
+    padding-top: 20px;
+  }
+
+  .header-menu a {
+    text-decoration: none;
+    color: var(--secondary_1);
+    margin-right: 40px;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .desktop-menu {
+    display: flex;
+    width: 100%;
+    justify-content: right;
+    background-color: var(--secondary_1);
+    margin-top: 80px;
+    margin-bottom: 40px;
+    margin-left: 50px;
+  }
+
+  .desktop-menu a {
+    text-decoration: none;
+    margin-right: 40px;
+    color: var(--primary_4);
+    font-family: 'Lato', sans-serif;
+  }
+
+  .desktop-menu .os-logo {
+    margin-right: 450px;
+    font-size: large;
+    font-weight: bolder;
+  }
+
+  .desktop-menu .fab {
+    color: var(--primary_2);
+    font-size: large;
+    font-weight: bolder;
+  }
+
+  .desktop-menu .about {
+    color: var(--primary_2);
+  }
+
+  .desktop-menu .OS-campaign {
+    color: var(--primary_2);
+    border: solid 3px var(--primary_2);
+    padding: 7px;
+  }
+
+  nav {
+    position: fixed;
+    width: 100%;
+    overflow: hidden;
+    top: 0;
+    z-index: 600;
+  }
+
+  #about-first-page {
+    padding-top: 60px;
+  }
+
+  .first-section-background {
+    background-color: #f3ebeb;
+  }
+
+  .about-mobile-first-section h1 {
+    font-size: 40px;
+  }
+
+  .about-mobile-first-section h2 {
+    color: var(--secondary_5);
+    font-size: 70px;
+  }
+
+  .about-mobile-first-section h2::after {
+    content: "";
+    background-color: var(--primary_2);
+    display: block;
+    margin: 0 auto;
+    width: 1.3em;
+    height: 2px;
+    border-radius: 0.5px;
+    margin-top: 0.5em;
+  }
+
+  .about-mobile-first-section p {
+    margin: 5% auto;
+    background-color: var(--secondary_1);
+    border: 2px solid var(--secondary_4);
+    color: var(--primary_4);
+    padding: 50px;
+    line-height: 25px;
+    width: 60%;
+  }
+
+  .about-mobile-first-section h3 {
+    font-size: 25px;
+    text-align: center;
+  }
+
+  .about-mobile-first-section h4 {
+    font-size: 18px;
+    color: #202125;
+  }
+
+  .about-second-section {
+    padding-top: 40px;
+  }
+
+  .about-second-section h3::after {
+    content: "";
+    background-color: var(--primary_2);
+    display: block;
+    margin: 0 auto;
+    width: 1.5em;
+    height: 2px;
+    border-radius: 0.5px;
+    margin-top: 0.5em;
+  }
+
+  .logo-2 {
+    font-family: 'Lato', sans-serif;
+    font-weight: 900;
+    font-size: 40px;
+    justify-content: center;
+    border: 1px solid var(--primary_5);
+    padding: 25px 10%;
+    margin: 3% 0;
+    text-align: center;
+  }
+
+  .previous-event {
+    display: flex;
+    flex-direction: row;
+    width: 50%;
+    margin-left: 350px;
+  }
+
+  .partners {
+    display: none;
+  }
+
+  .mobile-about-footer {
+    background-color: #dfd6d5;
+  }
```

Comment:
- [ ] As requested by the previous code reviewer, you should consider making this background a dark color, for the desktop version.